weather-app/src/app/current-weather/current-weather.component.html

```diff
@@ -3,11 +3,12 @@
     but only visible once it gets response.
  -->
 <div *ngIf="response">
-    <h2>City: {{ response.name}}</h2>
+    <h2>City: {{ response.name }}</h2>
+    <img [src]="imagePath" alt="picture not available">    
```

Comment:
Please change alt attribute to more descriptive value

Reply:
Hi @mlecic ,

Thank you for you observation. Alt attribute has been changed.